packages/token-balance-cache/src/tbc.ts

```diff
@@ -80,7 +83,7 @@ export class TokenBalanceCache extends JobRunner<ICache> implements ITokenBalanc
     // otherwise, return bps that support node's base
     // TODO: ensure all nodes have proper bases / balance types in db...
     const node = this._nodes[nodeId];
-    if (!node) throw new Error('Could not find node');
+    if (!node) this.statsDSender.sendAndThrowError(new Error('Could not find node'));
```

Comment:
Not a big fan of this pattern -- ideally we should send errors on catch, rather than bundling error throwing and sending. Any particular reason to prefer this over that?

Reply:
I think you are right that bundling error throwing and sending is not a good idea. 

My reasoning behind sending the error from within TBC is that it would reduce code coupling. If we instead send errors on catch, it will require us to access TbcStatsDSender through the calling classes.

But sending in catch is fine with me.